docs/tutorial/part-four/index.md

```diff
@@ -354,26 +336,20 @@ export default ({ children }) => {
 // highlight-end
 ```
 
-Another success! 🎉
+एक और सफलता! 🎉
 
-![Page title and layout title both pulling from siteMetadata](site-metadata-two-titles.png)
+![पेज शीर्षक और लेआउट शीर्षक दोनों siteMetadata से पुल कर रहे हैं](site-metadata-two-titles.png)
 
-Why use two different queries here? These examples were quick introductions to
-the query types, how they are formatted, and where they can be used. For now,
-keep in mind that only pages can make page queries. Non-page components, such as
-Layout, can use StaticQuery. [Part 7](/tutorial/part-seven/) of the tutorial explains these in greater
-depth.
+यहां दो अलग-अलग queries का उपयोग क्यों करें? ये उदाहरण त्वरित परिचय थे query प्रकार, उन्हें कैसे स्वरूपित किया जाता है, और कहाँ उनका उपयोग किया जा सकता है। अभी के लिए ध्यान रखें कि केवल पेज queries बना सकते हैं। गैर-पेज कौम्पोनॅन्टस, जैसे कि लेआउट, StaticQuery का उपयोग कर सकते हैं। [भाग ७](/tutorial/part-seven/) ट्यूटोरियल मे इसकी गहराई से अधिक व्याख्या की गयी है।
```

Comment:
`७` => `7`

lets keep numbers as it is. everyone does not understands numbers in hindi